.github/actions/nm-get-docker-tags/action.yml

```diff
@@ -22,15 +22,15 @@ runs:
   steps:
   - id: tags
     run: |
-      BUILD_VERSION=`echo "${{ inputs.wheel }}" | cut -d'-' -f2`
+      BUILD_VERSION=`echo "${{ inputs.whl }}" | cut -d'-' -f2`
       if [[ "${{ inputs.wf_category }}" == "RELEASE" ]]; then
-          TAG="v${build_version}"
+          TAG="v${BUILD_VERSION}"
           EXTRA_TAG=latest
```

Comment:
should the EXTRA_TAG value be in quotes?

Reply:
This should work fine in bash.